frappe/public/js/frappe/web_form/webform_script.js

```diff
@@ -115,7 +115,11 @@ frappe.ready(function() {
 				if (["Attach", "Attach Image"].includes(df.fieldtype)) {
 					if (!df.options)
 						df.options = {};
-					df.options.disable_file_browser = true;
+					try{
```

Comment:
@dinukasal Can you post the traceback of the error so that the actual issue can be fixed?